packages/@aws-cdk/aws-apigateway/lib/integrations/stepfunctions.ts

```diff
@@ -0,0 +1,314 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import * as iam from '@aws-cdk/aws-iam';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import { Token } from '@aws-cdk/core';
+import { RequestContext } from '.';
+import { IntegrationConfig, IntegrationOptions, PassthroughBehavior } from '../integration';
+import { Method } from '../method';
+import { AwsIntegration } from './aws';
+/**
+ * Options when configuring Step Functions integration with Rest API
+ */
+export interface StepFunctionsIntegrationOptions extends IntegrationOptions {
+  /**
+    * Action for the Step Functions integration. The list of supported API actions can be found
+    * on https://docs.aws.amazon.com/step-functions/latest/apireference/API_Operations.html
+    * @default 'StartSyncExecution'
```

Comment:
can we be smarter - pick the default based on the state machine type?

The state machine type is available - https://github.com/aws/aws-cdk/blob/024f18b0f4cb5aabed0be962ee01c1899fdf75b5/packages/%40aws-cdk/aws-stepfunctions/lib/state-machine.ts#L375

For imported state machines, we can just pick a default like it's already done here.

Reply:
not sure if this is valid as we changed the integration to be only Synchronous Express. Let me know if I misunderstood